modules/nf-core/parabricks/fq2bammeth/main.nf

```diff
@@ -32,6 +32,7 @@ process PARABRICKS_FQ2BAMMETH {
     def in_fq_command = meta.single_end ? "--in-se-fq $reads" : "--in-fq $reads"
     def known_sites_command = known_sites ? known_sites.collect{"--knownSites $it"}.join(' ') : ""
     def known_sites_output = known_sites ? "--out-recal-file ${prefix}.table" : ""
+    def num_gpus = task.accelerator ? "--num-gpus $task.accelerator.request" : ''
```

Comment:
This logic helps when testing on a local executor which has a GPU. `accelerator` is not supported by local executor.